shopify_python/google_styleguide.py

```diff
@@ -63,6 +63,9 @@ class GoogleStyleGuideChecker(checkers.BaseChecker):
                   'finally-too-long',
                   "The larger the 'finally' body size, the more likely that an exception will be raised during "
                   "resource cleanup activities."),
+        'C2610': ('%s imports multiple items in one import statement',
```

Comment:
Specify `args` in the message if you want formatted output; currently it shows:

```
C:  8, 0: %s imports multiple items in one import statement (multiple-import-items)
```

Perhaps reword it to read:
```
Multiple items imported from %(module)s in one import statement.
```

And in `add_message` provide `args={'module': node.modname}`.